### PR TITLE
Add Python OpenTelemetry example app

### PR DESCRIPTION
**Pull request by tombruijn**
Use the AppSignal standalone agent Docker image (private at this time).

I haven't added it to the CI build yet as it uses a private image. We can add it later when it's public.

### CODE REVIEW

**Comment by unflxw** [comment]
> the AppSignal standalone agent Docker image (private at this time)

Is there a reason we don't want to make those public? I don't know if @shairyar has access to our Docker Hub account.

File: python/django-4/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+```
+docker-compose up
+```
+
+## Setup
+
+```
+python manage.py makemigrations
+python manage.py migrate
+```
```
Comment (unflxw) [inline python/django-4/README.md:20]: Is this README still relevant? It seems to me that these instructions are no longer necessary (`rake app=python/django-4 app:up` should do all this, via `commands/run` and `processmon`)

**Review by unflxw — COMMENTED** [review]
I haven't tried running it yet -- just leaving some comments here.

Would it be okay to rename it to `django4-celery` or `django-celery`? It's a test setup for _both_, really, and we follow that naming convention for other test setups (`plug-oban` or `rails6-sidekiq`)

**Comment by tombruijn** [comment]
> Is there a reason we don't want to make those public? I don't know if @shairyar has access to our Docker Hub account.

We can make it public once the PR for it is merged. https://github.com/appsignal/appsignal-agent/pull/932

**Comment by tombruijn** [comment]
> Would it be okay to rename it to django4-celery or django-celery? It's a test setup for both, really, and we follow that naming convention for other test setups (plug-oban or rails6-sidekiq)

I've renamed the app and removed the extra README file.

File: python/django4-celery/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    image: postgres
+    ports:
+      - "5432:5432"
+    env_file:
+      - postgres.env
+  redis:
+    image: "redis:6.2.7"
+    ports:
+      - "6379:6379"
+  appsignal:
```
Comment (unflxw) [inline python/django4-celery/docker-compose.yml:14]: nit: maybe let's call this one "agent"?

**Pull request merged**